README.md

```diff
@@ -269,20 +269,45 @@ NOTE: Windows is not yet supported
 It is possible to build io.js with
 [OpenSSL FIPS module](https://www.openssl.org/docs/fips/fipsnotes.html).
 
+<B>Note</B> that building in this way does NOT allow you to
```

Comment:
To be consistent with other parts of this file, using `**` would be better here I think